src/plugins/intel_cpu/src/transformations/transformation_pipeline.cpp

```diff
@@ -408,12 +409,15 @@ void Transformations::PreLpt(const std::vector<ov::element::Type>& defaultPrecis
         type_to_fuse_map fuse_map = {{ov::op::PagedAttentionExtension::get_type_info_static(), fuse_type_to_pa}};
 #endif
         const bool keep_precision_sensitive_in_fp32 = true;
+        const bool need_convert_input_output_precision = false;
+        const bool save_original_precision_attribute = true;
         CPU_REGISTER_PASS_COMMON(manager,
                                  ov::pass::ConvertPrecision,
                                  fp_convert_precision_map,
                                  fuse_map,
                                  keep_precision_sensitive_in_fp32,
-                                 false);
+                                 need_convert_input_output_precision,
+                                 save_original_precision_attribute);
```

Comment:
Hi, @xczhai : not a review comments, but a question: from the description of this 'save_original_precision_attribute' param, it seems used to save the original precision of this node. but I didn't  find any change of such original precisions process in this pr, so which part of logic will be effected based on this change?

Reply:
Same question here, where it is actually being used and what logic does it affect?

Reply:
Also, an original name `store_original_precision_as_rt_attribute` explains the idea behind this flag better.

Reply:
@liubo-intel @EgorDuplensky 
answer your question.
1. `store_original_precision_as_rt_attribute` will work in the line https://github.com/openvinotoolkit/openvino/blob/master/src/common/transformations/src/transformations/convert_precision.cpp#L426. It means that such pass will insert a `Convert` node after `ReadValue`. But it is not necessary in f16 hint. As a result, the following pass `sdpafusion` cannot match because such extra `Convert`.
2. Both CPU and GPU call `ConvertPrecision` to impl `f16` conversion. In most cases, the output graph should be similar. https://github.com/openvinotoolkit/openvino/blob/1b3550ecdc9ab9d33c4936d671d33de4ff1f7bd3/src/plugins/intel_gpu/src/plugin/transformations_pipeline.cpp#L394. It is aligned with `GPU`. I think it is good f16 reference since gpu starts f16 earlier than cpu.